### PR TITLE
Bedes fix

### LINKED ISSUE #154: Fix measure search on BuildingSync website
When searching for text within a measure, the results don't actually filter.

To recreate: 

* go here: https://buildingsync.net/measures/2.4.0/
* Add in 'water' to the search box
* No filtering...

### PR DESCRIPTION
**Pull request by kflemin**
- update bedes mappings
- small css tweak on schema page
- added words next to searchbox on measures page - fixes #154 
- added buildingsync google group

### CODE REVIEW

File: bsyncviewer/templates/enumerations.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <form class="form-inline">
           <div class="form-group mb-2">
             <label for="search-input">Search:</label>
-            <input type="text" class="form-control" id="search-input" oninput="updateSearch()" placeholder="Keywords">
+            <input type="text" class="form-control" id="search-input" oninput="updateSearch()" placeholder="Keywords"> (type to search, press Enter to clear)
```
Comment (nllong) [inline bsyncviewer/templates/enumerations.html:37]: 👍

**Review by nllong — APPROVED** [review]
does this address the new mapping issues that @JieXiong9119 and @haneslinger were working on?

Thanks for fixing the CSS and search language!

File: bsyncviewer/templates/contact.html [hunk 2]
```diff
@@ -8,9 +8,14 @@
 
 <div class="container bsync-main">
 	<p class="big-margins">
-		Please direct any questions to <a href="mailto:BuildingSync@ee.doe.gov">BuildingSync@ee.doe.gov</a>. Also, you can follow us on GitHub as we make proposals, changes, and new releases.
+		Please direct any questions to <a href="mailto:BuildingSync@ee.doe.gov">BuildingSync@ee.doe.gov</a>. Also, you can follow us on <a href="https://github.com/BuildingSync" target="_blank">GitHub</a> as we make proposals, changes, and new releases.
 	</p>
 
+	<p class="big-margins">
+		To subscribe to BuildingSync updates send a blank email to 
+    <strong>buildingsync+subscribe@googlegroups.com</strong> or visit the
```
Comment (nllong) [inline bsyncviewer/templates/contact.html:16]: ❤️ most excellent.

File: bsyncviewer/lib/bedes/v2.5/schema2.4.0/bedes-mappings-enumerations.csv [hunk 3]
```diff
@@ -953,10 +953,10 @@ therms/h,40109,0534d11a-ec7b-47fb-a23a-c4739ae0ca79,therms/hr,therms per hour,ht
...
-Other,40114,1bf118dd-9a9e-440c-8e8f-b3a92ab62855,Other,Other,https://bedes.lbl.gov/node/1bf118dd-9a9e-440c-8e8f-b3a92ab62855,1d87e8ff-747e-4a30-8342-e43b96e53bd4,1,49215
-Roof,40115,fe6a5ef0-9365-4932-bed7-eb0ca9df5c0b,Roof,Makes up theÂ top exterior boundary of the premises envelope.Â It isÂ generally horizontal or sloped less than 60 degrees from horizontal. Represents the complete roof construction.,https://bedes.lbl.gov/node/fe6a5ef0-9365-4932-bed7-eb0ca9df5c0b,686035b9-341d-490b-ad08-b5c192b701b1,1,49222
-Mechanical Room,40116,cd0e8b58-59bb-4e6c-b153-301be7e32785,Mechanical room,The mechanical room is dedicated to the mechanical equipment and its associated electrical equipment.,https://bedes.lbl.gov/node/cd0e8b58-59bb-4e6c-b153-301be7e32785,c96589bd-0dfc-47a4-b040-11dba49d2ebd,0.973333,49222
+Cooling ton,40113,437f592b-2605-4be4-bffc-debf6de305d4,cooling tons,cooling tons,https://bedes.lbl.gov/node/437f592b-2605-4be4-bffc-debf6de305d4,10645e3b-270a-4251-8153-36017540d539,0.99,49215
+Other,40114,1bf118dd-9a9e-440c-8e8f-b3a92ab62855,Other,Other,https://bedes.lbl.gov/node/437f592b-2605-4be4-bffc-debf6de305d4,1d87e8ff-747e-4a30-8342-e43b96e53bd4,1,49215
```
Comment (JieXiong9119) [inline bsyncviewer/lib/bedes/v2.5/schema2.4.0/bedes-mappings-enumerations.csv:957]: I must have made a selection mistakes. The lines after `Cooling ton` should not be changed.

Comment (nllong) [reply bsyncviewer/lib/bedes/v2.5/schema2.4.0/bedes-mappings-enumerations.csv:957]: doh, can you create a PR with the fix then.

Comment (JieXiong9119) [reply bsyncviewer/lib/bedes/v2.5/schema2.4.0/bedes-mappings-enumerations.csv:957]: #159

**Pull request merged**